sonoff/xsns_91_tx20.ino

```diff
@@ -0,0 +1,204 @@
+/*
+  xsns_91_Tx20.ino - Tx20 wind sensor
+  support for Sonoff-Tasmota
+
+  Copyright (C) 2018  Thomas Eckerstorfer and Theo Arends
+  based on https://github.com/bunnyhu/ESP8266_TX20_wind_sensor/
+  http://blog.bubux.de/windsensor-tx20-mit-esp8266/
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+/*********************************************************************************************\
+ * Tx20 wind sensor
+ \*********************************************************************************************/
+
```

Comment:
You need to add a 
```suggestion
#ifdef USE_TX20_wind_sensor
```
in order to enable or disable this sensor from user_config.h file

By default this sensor should be disabled in user_config.h